pygenometracks/tracks/HiCMatrixTrack.py

```diff
@@ -311,3 +312,44 @@ def pcolormesh_45deg(self, ax, matrix_c, start_pos_vector, vmin=None, vmax=None)
         im = ax.pcolormesh(x, y, np.flipud(matrix_c),
                            vmin=vmin, vmax=vmax, cmap=self.cmap, norm=self.norm)
         return im
+
+    def process_colormap(self):
+        if 'colormap' not in self.properties:
+            return(DEFAULT_MATRIX_COLORMAP)
+            # If someone what to use its own colormap, 
+            # he can specify the rgb values or color values:
+            # For example:
+            # colormap = ['white', (1, 0.88, 2./3), (1, 0.74, 0.25), (1, 0.5, 0), (1, 0.19, 0), (0.74, 0, 0), (0.35, 0, 0)]
+        elif self.properties['colormap'][0] == '[':
```

Comment:
wondering if you need need to `stip()` the self.properties['colormap'][0] before.

Reply:
For me the parser do the job.
I checked with 10 spaces between the equal and the [ and it worked...